rtc/configuration.md

```diff
@@ -304,46 +359,65 @@ This option allows applications to show when a user enters the RTC session. When
 
 Only one `rtc_client_connected` event will be fired per client connection. Connecting to a session with multiple existing clients will fire separate `rtc_client_connected` events for each existing client.
 
-To help with generating a user interface for connected users, four pieces of data are provided:
+**Type:** `Function`
+
+**Required:** no
+
+#### Input fields for `rtc_client_connected`
+
+To help with generating a user interface for connected users, an object with multiple fields is provided.
 
 #### User ID
 
+| Field | Type |
+|-------|:----:|
+| `userId` | `string` |
+
 This is the user's unique ID (the `sub` field from their [JWT](({{site.baseurl}}/rtc/jwt-authentication/)), which is also used for [`rtc_user_details_provider`](#rtc_user_details_provider)). Multiple connection events will be received with the same user ID if a user opens multiple sessions (for example on desktop and mobile).
 
 #### User Details
 
+| Field | Type |
+|-------|:----:|
+| `userDetails` | `object` |
+
 This is a copy of the object returned by [`rtc_user_details_provider`](#rtc_user_details_provider). RTC only uses the `fullName` field, but the entire object will be cloned and passed to `rtc_client_connected`.
 
+If a user details provider is not configured, this will be an empty object.
+
 #### Client ID
 
+| Field | Type |
+|-------|:----:|
+| `clientId` | `string` |
+
 This is a unique identifier, generated by the RTC protocol, that can be used to differentiate between the same user connecting multiple times.
 
 #### Caret Number
 
-This number corresponds to one of [the 8 colours defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
+| Field | Type |
+|-------|:----:|
+| `caretNumber` | `integer` |
 
+This will be a number between 1 and 8, corresponding to one of [the 8 colours defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
```

Comment:
```suggestion
This will be a number between 1 and 8, corresponding to one of [the 8 colors defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
```